react/components/ProductImages/components/BlurredLoader/index.js

```diff
@@ -154,7 +154,7 @@ class BlurredLoader extends React.Component {
               minRatio={imageMinRatio}
               className={`absolute z-1 w-100 center left-0 right-0 bottom-0 top-0 ${
                 styles.imageTransitionOpacity
-              } db ${loadingClass}`}
+              } db`}
```

Comment:
is `loadingClass` being used anywhere else?

Reply:
Yeah, @lbebber. Line [149](https://github.com/vtex-apps/store-components/pull/422/files#diff-34fd93291dd7c0b385666821021bca20R149)